pkg/crd/generator/testData/pkg/apis/fun/v1alpha1/toy_types.go

```diff
@@ -50,7 +51,7 @@ type ToySpec struct {
 	Template v1.PodTemplateSpec       `json:"template"`
 	Claim    v1.PersistentVolumeClaim `json:"claim,omitempty"`
 
-	Replicas *int32 `json:"replicas"`
+	Replicas *intstr.IntOrString `json:"replicas"`
```

Comment:
The previous field-type was for testing *int32, so can we retain the previous type and introduce new field for testing `IntOrString` ?

Reply:
Done!